cabal.project.local

```diff
@@ -0,0 +1,9 @@
+with-compiler: /opt/ghc/8.4.1/bin/ghc
```

Comment:
I don't think that's a good idea to push `cabal.project.local`, it's called `local` for a reason, it must be in `.gitignore`. Instead you could name this file `cabal.project.local.ubuntu-example`, and then `cp` in the Travis script.

Reply:
@int-index Okay, I can do `cp`. I tried to generate it with `echo` but failed :disappointed: 
If it passes, renaming I can do.